test/FastTests/Client/RavenCommandTest.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Raven.Client.Http;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace FastTests.Client
+{
+    public class RavenCommandTest : RavenTestBase
+    {
+        public RavenCommandTest(ITestOutputHelper output) : base(output)
+        {
+        }
+        
+        [Fact]
+        public void WhenCommandCanBeCheckedForFastestNode_ItCanRunInParallel()
+        {
+            var expected = new[]
+            {
+                "GetClusterTopologyCommand", "GetDatabaseTopologyCommand", "GetNodeInfoCommand", "GetRawStreamResultCommand", "GetTcpInfoCommand",
```

Comment:
can we scan this using reflection - list is good for today - but it can easily be out of sync tomorrow.